spec/lib/avalon/batch_ingest_spec.rb

```diff
@@ -114,14 +119,16 @@
     end
 
     it 'should ingest batch with spaces in name' do
-      space_batch_path = File.join('spec/fixtures/dropbox/example batch ingest', 'batch manifest with spaces.xlsx')
+      FileUtils.cp_r 'spec/fixtures/dropbox/example batch ingest', @dropbox_dir
```

Comment:
Do you need this change (and in the test below)?  I'd rather not create files during the tests since they don't always get cleaned up.